schema.graphql

```diff
@@ -478,6 +478,33 @@ type ProjectMinterConfiguration @entity {
   maxInvocations: BigInt
 }
 
+type Bid @entity {
+  "Unique identifier made up of for SEA - minter-project-bidder-value-timestamp-token [TODO - do something different from RAM]"
```

Comment:
I think since we have token in this, we should remove project from this ID (since redundant).
Also, I'm thinking we should remove timestamp? Fine with indexing it as a field, but I don't think we need it for uniqueness, and just adds more info we need to be able to properly load a token, right?
thoughts?

Reply:
Agreed on the project id! I'll remove that. I think we need to keep timestamp concatenated in the ID for the case of a user bidding -> being outbid -> bidding again with a higher value for the same token ID. That way we have a full historical record of all bids and not just their latest bid.

Reply:
I think in any case, since the bidding -> being outbid -> bidding again can technically happen in a single block, timestamp still doesn't add any uniqueness. However, doesn't including value protect against that case?

Reply:
Great point! - Right now it's `{minter}-{bidder}-{timestamp}-{token}` so I will update it to `{minter}-{bidder}-{value}-{token}`